moped-editor/src/views/projects/projectsListView/ProjectsListViewQueryConf.js

```diff
@@ -306,7 +301,7 @@ export const DEFAULT_HIDDEN_COLS = Object.entries(
   PROJECT_LIST_VIEW_QUERY_CONFIG.columns
 ).reduce((acc, [columnName, config]) => {
   if (config.showInTable === true) {
-    acc[columnName] = config.defaultHidden;
+    acc[columnName] = !config.defaultHidden;
```

Comment:
https://mui.com/x/react-data-grid/column-visibility/

Data Grids default is all columns visible, setting it false in the column visibility model hides the column.